### DISCUSSION TITLE
Octofarm in docker for arm 32bit

### DISCUSSION THREAD

**Discussion by rsurgiewicz** [OctoFarm/OctoFarm#573] [General]
Hi there,

kudos for the great tool!

I just wanted to let You know I found a way to run octofarm on regular rpi amrv7 raspbian 32 bit.
32bit mongo is necessary

Here is my repo with Dockerfiles & docker-compose.yml
https://github.com/rsurgiewicz/octofarm-docker-armv7

Works fine (but only in dev mode/ no pm2)
![chrome_5acL04nhgP](https://user-images.githubusercontent.com/47541623/118181675-4ee91b00-b438-11eb-9a83-92c6bd2b2b0e.png)

Let me know if it helps anyone there.

Best

**Comment by NotExpectedYet** [comment]
> 
> 
> Hi there,
> 
> kudos for the great tool!
> 
> I just wanted to let You know I found a way to run octofarm on regular rpi amrv7 raspbian 32 bit.
> 32bit mongo is necessary
> 
> Here is my repo with Dockerfiles & docker-compose.yml
> https://github.com/rsurgiewicz/octofarm-docker-armv7
> 
> Works fine (but only in dev mode/ no pm2)
> ![chrome_5acL04nhgP](https://user-images.githubusercontent.com/47541623/118181675-4ee91b00-b438-11eb-9a83-92c6bd2b2b0e.png)
> 
> Let me know if it helps anyone there.
> 
> Best

Oh nice! There's a 32-bit mongo? I had no idea. 

Doesn't the alpine image work on Pi? I presumed it did but will check out your repo over the weekend.

**Reply by NotExpectedYet** [reply]
Also thanks for your kind words!

**Reply by rsurgiewicz** [reply]
Hi @NotExpectedYet ,
well none of images (standard,alpine,monolithic) worked for me. 
I've tried on different rpis but all gave
`exec user process caused “exec format error”`

Thanks for the reply :)

**Reply by NotExpectedYet** [reply]
No worries :) 

Sorry got way busy and forget to check this. I'll allot some time through the week.
